compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/tree/SyntaxKind.java

```diff
@@ -77,12 +77,16 @@
     PERCENT_TOKEN(556, "%"),
     ASTERISK_TOKEN(557, "*"),
     LT_TOKEN(558, "<"),
-    EQUAL_LT_TOKEN(559, "<="),
+    LT_EQUAL_TOKEN(559, "<="),
     GT_TOKEN(560, ">"),
     EQUAL_GT_TOKEN(561, "=>"),
     QUESTION_MARK_TOKEN(562, "?"),
     PIPE_TOKEN(563, "|"),
-
+    GT_EQUAL_TOKEN(564, ">="),
+    EXCLAMATION_MARK_TOKEN(565, "!"),
+    EM_EQUAL_TOKEN(566, "!="),
+    EM_DOUBLE_EQUAL_TOKEN(567, "!=="),
```

Comment:
Shall we rename this to `NOT_EQUAL` and `NOT_DOUBLE_EQUAL`?

Reply:
sure.